crates/starknet-devnet-core/src/state/mod.rs

```diff
@@ -59,27 +59,41 @@ pub trait CustomState {
     ) -> DevnetResult<()>;
 }
 
-#[derive(Default, Clone)]
+#[derive(Clone)]
 /// Utility structure that makes it easier to calculate state diff later on. Classes are first
 /// inserted into the staging area (pending state), later to be committed (assigned a block number
 /// to mark when they were added). Committed doesn't necessarily mean the class is a part of the
 /// latest state, just that it is bound to be. Since there is no way of telling if a class is in the
 /// latest state or no, retrieving from the latest state has to be done via block number.
-pub struct CommittedClassStorage {
-    staging: HashMap<ClassHash, ContractClass>,
-    committed: HashMap<ClassHash, (ContractClass, u64)>,
+pub struct CommittedClassStorage<TClassHash, TContractClass> {
```

Comment:
As discussed, generics can be removed.

Reply:
[3517439](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/633/commits/3517439fa746fa3299a81b93af05cf74cf2c1286)